composer.json

```diff
@@ -28,8 +28,9 @@
 	},
 	"scripts": {
 		"lint": "vendor/bin/phpcs --extensions=php --standard=./ruleset.xml",
-		"test": "phpunit",
+		"test": "XDEBUG_TRIGGER=1 phpunit",
 		"test:install": "bash ./bin/install-wp-tests-local.sh",
+		"test:xdebug-setup": "bash ./bin/setup-phpunit-xdebug.sh",
```

Comment:
Is it really related to the tests? or is it just script to install xdebug?

Maybe just `setup-xdebug`